communitiesuk/funding-service-design-post-award-data-store
# Instantiate data via ingest API call

https://trello.com/c/YOSvTOOJ
https://trello.com/c/ltVRaRu2

### Change description
Improved the "workflow" so that API ingest call actually loads data into fake DB. This data can then be queried via the usual get calls for project or package.

Previously all returned data was instantiated at import in a "hacky" indirect way. Removing/ uncoupling this unblocks more suitable end to end testing etc. 

We also now no longer store live data spreadsheets in the repo, these can be ingested via the UI from local sources.



### How to test

- run the flask debug server locally
- on swagger UI, ingest a spreadsheet via the ingest call:
- The Data Model project spreadsheet will probably fail and should return lots of useful validation messages. 
- The basic example spreadsheet in the test resources folder should load successfully
- Once ingested, run the get package or get project data calls with valid input ID's from the spreadsheet - you should get a payload back in the response containing all the data.
